README.md

```diff
@@ -1,29 +1,45 @@
-# Create React Express App
+# Pub Trivia App
 
-## About This Boilerplate
+## Overview
 
-This setup allows for a Node/Express/React app which can be easily deployed to Heroku.
+In these days of families and friends being spread out across wide geographical locations, and even more in our current coronavirus crisis, it is sometimes hard to be able to get together to share some fun. This is where our app comes in. We are creating a virtual pub quiz (bring your own beer!) where you can login and invite friends and family to join you for a quiz! You play against each other in real time, and watch your avatars race each other to be the winner. After the quiz your friends might want to sign up too and create a group, so you can keep competing again and again. When you're done playing why not contribute a question yourself to bamboozle someone else. 
```

Comment:
@dropcat13 this is great! One small tweak, I think we decided the group thing would be a stretch goal so maybe we don't want to include this sentence:

> After the quiz your friends might want to sign up too and create a group, so you can keep competing again and again.